userspace/libscap/scap.c

```diff
@@ -337,6 +340,29 @@ scap_t* scap_open_live_int(char *error, int32_t *rc,
 		return NULL;
 	}
 
+	if (syscalls_of_interest)
+	{
+		memcpy(handle->syscalls_of_interest, syscalls_of_interest->syscalls, sizeof(bool) * SYSCALL_TABLE_SIZE);
+		// UF_NEVER_DROP syscalls must be always traced
+		for (int i = 0; i < SYSCALL_TABLE_SIZE; i++)
+		{
+			if (g_syscall_table[i].flags & UF_NEVER_DROP)
```

Comment:
So this means that by default *only* syscalls with the UF_NEVER_DROP will be passed up by the kernel? If true, that seems pretty strict. Why not *exclude* those with EF_DROP_SIMPLE_CONS instead? That's a wider set but matches what falco users will expect to see.

Reply:
Nope, by default all syscalls are interesting, as you can see here: https://github.com/falcosecurity/libs/pull/115/files/ffb00b95487e247c5f1447b2829b9d3d8033aaec#diff-633539375239ac63e4a9e3358f0852d437b22e9c9f2c96de5e04e346b0b4784fR496